kpi/deployment_backends/kobocat_backend.py

```diff
@@ -401,11 +407,11 @@ def set_asset_uid(self, force=False):
 
     def set_has_kpi_hooks(self):
         """
-        PATCH `has_kpi_hooks` boolean of survey.
-        It lets KoBoCAT know whether it needs to ping KPI
+        `PATCH` `has_kpi_hooks` boolean of related KoBoCAT XForm.
```

Comment:
i respect your consistency, but in the future i don't think wrapping HTTP methods in backticks is really necessary (and it gets kind of messy-looking in contexts like this)

Reply:
😭